helpers/bootrr-generic-tests

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+assert_file_is_empty deferred-probe-empty /sys/kernel/debug/devices_deferred
```

Comment:
is that clearly stated/known that /sys/kernel/debug is required/mounted for bootrr?

Reply:
I ignore if /sys/kernel/debug is required. Someone knows?
If not, I can probably check it and skip test that depend on it.

Reply:
Yes, it is. So probably is worth check first if /sys/kernel/debug exists and if not just signal lava to skip the test?

Reply:
In the scope of assert_file_is_empty, I test first if the file exists and return skip if not